src/Momento.Sdk/Config/Configuration.cs

```diff
@@ -8,20 +8,39 @@
 
 namespace Momento.Sdk.Config;
 
+/// <inheritdoc cref="Momento.Sdk.Config.IConfiguration" />
 public class Configuration : IConfiguration
 {
+    /// <inheritdoc cref="Microsoft.Extensions.Logging.ILoggerFactory" />
```

Comment:
I like this, if it works the way we want it to.  @malandis is this a reasonable thing to do?  Annotating a Property with an inheritdoc for a Class?